contracts/HydraChain/modules/PowerExponent/PowerExponent.sol

```diff
@@ -1,58 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.17;
-
-import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
-
-import {IPowerExponent, PowerExponentStore} from "./IPowerExponent.sol";
-
-abstract contract PowerExponent is IPowerExponent, Ownable2StepUpgradeable {
-    PowerExponentStore public powerExponent;
-
-    // _______________ Initializer _______________
-
-    function __PowerExponent_init() internal onlyInitializing {
-        __PowerExponent_init_unchained();
-    }
-
-    function __PowerExponent_init_unchained() internal onlyInitializing {
-        powerExponent = PowerExponentStore({value: 5000, pendingValue: 0});
-    }
-
-    // _______________ External functions _______________
-
-    /**
-     * @inheritdoc IPowerExponent
-     */
-    function getExponent() external view returns (uint256 numerator, uint256 denominator) {
```

Comment:
wouldn't it be good to keep this function?

Reply:
no, because now we keep the numerator only and it is easily retrievable as it is public variable